tests/features/logIn.spec.ts

```diff
@@ -2,16 +2,36 @@ import { test, expect, Page } from '@playwright/test';
 import LoginPage from '../poms/LoginPage/LoginPage';
 import MainDashboard from '../poms/MainDashboard/MainDashboard';
 import { USER_NAME, USER_PWD } from '../data';
-
+import { pwdLabelTxt, userNameLabelTxt, wrongCredentialsError } from '../../constants';
 
 test.describe('Log in test suite', () => {
     test('Log in to the app', async ({page}) => {
         const loginPage = new LoginPage(page);
         const mainDashboard = new MainDashboard(page);
 
         await loginPage.goTo();
+        await expect(loginPage.userNameLabel).toHaveText(userNameLabelTxt);
+        await expect(loginPage.pwdLabel).toHaveText(pwdLabelTxt);
         await loginPage.logIn(USER_NAME, USER_PWD);
 
         await mainDashboard.pageLoaded();
     } )
+
+    test('Try to log in with a wrong password', async ({page}) => {
+        const loginPage = new LoginPage(page);
+
+        await loginPage.goTo();
```

Comment:
I would suggest to move this part to beforeEach hook since you are repeating this in each case. Apply this change in all 3 test cases.